src/widget/condaStoreWidget.tsx

```diff
@@ -12,7 +12,10 @@ interface IJlabAppProps extends IAppProps {
 class JlabApp extends App<IJlabAppProps> {
   constructor({ settings }: IJlabAppProps) {
     super({
-      pref: settings.composite as IAppProps['pref'],
+      pref: {
+        ...(settings.composite as IAppProps['pref']),
+        routerType: 'memory'
```

Comment:
`tsc` doesn't like this: it's preventing this from building without a `// @ts-ignore`.

Reply:
Should be good now with the update to conda-store-ui 2024.10.1, right?